Header header

float32[360] free_dist_from_ground
float32[360] free_dist_from_objects
float32[360] free_dist_combined